Umpah-iOS/Umpah-iOS/Source/Views/Components/Main/ExpandedStateView.swift

```diff
@@ -181,6 +224,12 @@ extension ExpandedStateView: UITableViewDataSource {
 
 // MARK: - UITableViewDelegate
 extension ExpandedStateView: UITableViewDelegate {
+    
+    func tableView(_ tableView: UITableView, heightForRowAt indexPath: IndexPath) -> CGFloat {
+        tableView.separatorStyle = state == .routine ? .none : .singleLine
+        return state == .routine ? 168 : UITableView.automaticDimension
```

Comment:
```suggestion
        tableView.separatorStyle = (state == .routine) ? .none : .singleLine
        return (state == .routine) ? 168 : UITableView.automaticDimension
```
괄호해주면 구분이 잘 되어서 코드가 더 잘 읽힐 거 같아요!